### ISSUE TITLE
freshchat icon is not displaying

### ISSUE THREAD

**Issue by Brad19** [BlueBambooStudios/gatsby-plugin-freshchat#2]
Hi,
I've included this plugin in my gatsby.config.js. But I could not see the icon displaying. Looks like when i inspect elements, the fc_frame has by default height and width as 0 and opacity is set to 0. Not sure what need to be done to get the widget displayed. Please advise

![image](https://user-images.githubusercontent.com/10935246/82984861-53090680-a036-11ea-9c40-ce96c3417c0c.png)

**Comment by haswalt** [comment]
Hi @Brad19 

It may be that something has changed in the way freshchat works, but this plugin simply injects the freshchat widget code into the page on render.

fc_frame is injected by their widget code, have you ensured that all setup has been completed on the freshchat control panel?

Are there any errors in the console?

**Comment by haswalt** [comment]
Also are you able to check that there is a script tag injected into the bottom of the page containing the init script for freschat. This will looking something like: 

```
<script>
window.fcWidget.init({
  token: "{YOUR_TOKEN_HERE}",
  host: "https://wchat.freshchat.com",
})
</script>
```

**Comment by Brad19** [comment]
@haswalt Thanks for your response. I found the root cause. The reason behind not showing was due to the Token which has JWT enabled. So an anonymous user will never be able to see the widget in public site. So we need a secured token now to be passed. But other thing that I want to ask is when I get the secured token from componentDidMount, I would need that token to get passed to fcWidge.init({...}). If you know how, Pls share it. I've started working towards this.
